oilandrope/settings.py

```diff
@@ -250,6 +250,19 @@
     ),
 }
 
+
+# Redis 
+# https://pypi.org/project/channels-redis/
+
+CHANNEL_LAYERS = {
```

Comment:
**Dude what the hell.**

This configuration is already set on `dev_settings.py`.

Also, `localhost` (a.k.a. 127.0.0.1) on production?
**I think not.**